orthoinference/src/main/java/org/reactome/orthoinference/PathwaysInferrer.java

```diff
@@ -168,40 +166,31 @@ private static void addInferredEventsToInferredPathways() throws Exception {
 	private static void inferPrecedingEvents() throws Exception
 	{
 		Set<GKInstance> seenPrecedingEvent = new HashSet<>();
-		for (GKInstance inferrableEventInst : updatedInferrableHumanEvents)
-		{
-			if (!seenPrecedingEvent.contains(inferrableEventInst))
-			{
-				if (inferrableEventInst.getAttributeValue(precedingEvent)!= null)
-				{
+		for (GKInstance inferrableEventInst : updatedInferrableHumanEvents) {
+			if (!seenPrecedingEvent.contains(inferrableEventInst)) {
+				if (inferrableEventInst.getAttributeValue(precedingEvent) != null) {
```

Comment:
Could these conditions be merged into one if-statement as `!seenPrecedingEvent.contains(inferrableEventInst) && inferrableEventInst.getAttributeValue(precedingEvent) != null` ?
I don't see an else-statement pairing with the inner if-statement so it seems like it should be OK, and would make the rest of this a little more readable.